src/imgtools/modules/segmentation.py

```
@@ -1,37 +1,175 @@
+# Applying a function to each label in the segmentation
+>>> def compute_statistics(label_image):
+>>>     return sitk.LabelStatisticsImageFilter().Execute(label_image)
+
+>>> stats = map_over_labels(segmentation=seg, f=compute_statistics)
+"""
+
+from __future__ import annotations
+
 import warnings
+from functools import wraps
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)